pkgs/development/compilers/sbcl/bootstrap.nix

```diff
@@ -23,7 +23,6 @@ let
       sha256 = "0sp5445rbvms6qvzhld0kwwvydw51vq5iaf4kdqsf2d9jvaz3yx5";
     };
     armv6l-linux = armv7l-linux;
-    x86_64-solaris = x86_64-linux;
```

Comment:
My impression was that this line was added only because nixpkgs evaluation on that target was broken due to missing option attribute.  E.g. see https://github.com/NixOS/nixpkgs/issues/20254 which is similar case.

Instead of adding wrong attributes for every possible target, it is now fixed by the assert.

Are you saying that sbcl is intended to work on x86_64-solaris?